core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -1367,6 +1367,10 @@ class EMConfig {
     @Min(1.0)
     var populationSize = 30
 
+    @Cfg("Fixed mutation rate")
```

Comment:
that is just repeating the name of the variable with spaces between words... need to provide some more info.